minerva-server/src/main/resources/GetAllModels.rq

```diff
@@ -0,0 +1,43 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#> 
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> 
+#model metadata
+PREFIX metago: <http://model.geneontology.org/>
+PREFIX lego: <http://geneontology.org/lego/>
+#PREFIX dc: <http://purl.org/dc/elements/1.1/>
+#PREFIX rdfs:<http://www.w3.org/2000/01/rdf-schema#> 
+#model data
+PREFIX part_of: <http://purl.obolibrary.org/obo/BFO_0000050>
+PREFIX occurs_in: <http://purl.obolibrary.org/obo/BFO_0000066>
+PREFIX enabled_by: <http://purl.obolibrary.org/obo/RO_0002333>
+PREFIX has_input: <http://purl.obolibrary.org/obo/RO_0002233>
+PREFIX has_output: <http://purl.obolibrary.org/obo/RO_0002234>
+PREFIX causally_upstream_of: <http://purl.obolibrary.org/obo/RO_0002411>
+PREFIX provides_direct_input_for: <http://purl.obolibrary.org/obo/RO_0002413>
+PREFIX directly_positively_regulates: <http://purl.obolibrary.org/obo/RO_0002629>
+
+SELECT  ?cam ?id ?date ?title ?state ?gp_class (GROUP_CONCAT(?contributor;separator=";") AS ?contributors)  
+WHERE {
+  
+  GRAPH ?cam {      
+      ?cam <http://purl.org/dc/elements/1.1/title> ?title ;
+           <http://purl.org/dc/elements/1.1/date> ?date ;
+           <http://purl.org/dc/elements/1.1/contributor> ?contributor ;
+           lego:modelstate ?state .          
+    	   optional { 
+             ?cam <http://www.geneontology.org/formats/oboInOwl#id> ?id 
+           }       	
+           #lpalbou: Baby Proofing the query since oboInOwl#id is not always there
+	  		BIND(IF(bound(?id), ?id, concat("gomodel:", substr(str(?cam), 31))) as ?id) .
```

Comment:
It seems like turning an IRI into a CURIE should happen somewhere else, not here.

Reply:
yup fixed